hibernation-setup-tool.c

```diff
@@ -106,9 +106,9 @@ static void log_impl(int log_level, const char *fmt, va_list ap)
     if (log_needs_prefix)
         printf("hibernation-setup-tool: ");
 
-    if (log_level & LOG_INFO)
+    if (log_level == LOG_INFO)
```

Comment:
The log level is a bitmask, so the original code is correct.  One can pass `LOG_INFO | LOG_ERR` and both will be set. It shouldn't make a difference in this program, though, because nobody should call `log_impl()` directly, but rather use `log_info()` or `log_fatal()` which sets only one each.

Reply:
Leaving with changed implementation